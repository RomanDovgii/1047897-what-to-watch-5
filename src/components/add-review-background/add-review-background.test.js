import React from "react";
import renderer from "react-test-renderer";
import AddReviewBackground from "./add-review-background";

const testBackgroundImage = `none`;
const testName = `none`;

it(
    `Should render correct AddReviewBackground`,
    () => {
      const tree = renderer
      .create(
          <AddReviewBackground
            name={testName}
            backgroundImage={testBackgroundImage}
          />
      )
      .toJSON();

      expect(tree).toMatchSnapshot();
    }
);
