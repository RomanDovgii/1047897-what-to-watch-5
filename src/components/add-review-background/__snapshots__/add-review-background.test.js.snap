// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render correct AddReviewBackground 1`] = `
<div
  className="movie-card__bg"
>
  <img
    alt="none"
    src="none"
  />
</div>
`;
